templates/glue-job-JSONToParquet.j2

```diff
@@ -40,7 +40,7 @@ Parameters:
   TimeoutInMinutes:
     Type: Number
     Description: The job timeout in minutes (integer).
-    Default: 120
+    Default: 720
```

Comment:
Do we have an idea of how long the total workflow time is now (with the 2nd round of processing data)?

Reply:
Is there a rationale behind the order or was the order random?

Reply:
> Do we have an idea of how long the total workflow time is now (with the 2nd round of processing data)?
 
Not sure, but some of the data types take longer than 2 hours to process. I needed to increase this time since I was getting timeout errors.